packages/openid4vc/src/openid4vc-holder/OpenId4vcSiopHolderService.ts

```diff
@@ -109,6 +110,14 @@ export class OpenId4VcSiopHolderService {
           challenge: nonce,
           domain: clientId,
           presentationSubmissionLocation: DifPresentationExchangeSubmissionLocation.EXTERNAL,
+          openid4vp: {
```

Comment:
how are these values mapped to the presentation? Ideally we don't add oid4vp specific properties to the pex service, but i can understand if it's not possible otherwise

Reply:
Yeah, I also don't like it, but I don't see an easy better way to propagete the values. There are used to calculate the session transcript

Reply:
okay and this is openid4vp specific, or is it mdoc specific?

Reply:
Hmm I guess depends on the viewpoint the data we need is expected to come for openid4vp, but we need it to create mdoc presentations.

Reply:
Okay so if we implement BLE exchange we wouldn't need these values?

Reply:
No we need it to compute the session transcript bytes. 

```
SessionTranscriptBytes = #6.24(bstr .cbor SessionTranscript)
SessionTranscript = [
    DeviceEngagementBytes,
    EReaderKeyBytes,
    Handover
]
DeviceEngagementBytes = #6.24(bstr .cbor DeviceEngagement)
Handover = QRHandover / NFCHandover
QRHandover= null
NFCHandover = [
    bstr        ; Binary value of the Handover Select Message
    bstr / null ; Binary value of the Handover Request Message,
; shall be null if NFC Static Handover was used DeviceEngagement is defined in 8.2.1.1.
]
```